### ISSUE TITLE
remove xformer

### ISSUE THREAD

**Issue by ydhongHIT** [baaivision/EVA#77]
Is it possible to set xattn=False in downstream task fine-tuning?

**Comment by Yuxin-CV** [comment]
Hi, `timm` has a EVA series implementation based on pytorch 2.0
https://huggingface.co/timm/eva02_large_patch14_448.mim_m38m_ft_in1k

**Comment by ydhongHIT** [comment]
> Hi, `timm` has a EVA series implementation based on pytorch 2.0 https://huggingface.co/timm/eva02_large_patch14_448.mim_m38m_ft_in1k

It seems that pytorch 2.0 adopts the same efficient implementation as xformer. Is it possible to directly use the naive implementation of self-attention for downstream task fine-tuning?

**Comment by ydhongHIT** [comment]
> > Hi, `timm` has a EVA series implementation based on pytorch 2.0 https://huggingface.co/timm/eva02_large_patch14_448.mim_m38m_ft_in1k
> 
> It seems that pytorch 2.0 adopts the same efficient implementation as xformer. Is it possible to directly use the naive implementation of self-attention for downstream task fine-tuning?

Regardless of GPU memory, whether using the naive implementation of self-attention for downstream task fine-tuning will affect accuracy?

**Comment by Yuxin-CV** [comment]
It is OK to use naive MHSA implementation
